### PR TITLE
Example of how a normal rdf graph (i.e. the bioregistry rdf) could be…

### PR DESCRIPTION
**Pull request by JervenBolleman**
… in the same store as this conversion service.

The idea would be that there is one bioregistry.io endpoint that does both this special conversion and normal queries on the rdf data of the bioregistry.

### CODE REVIEW

**Comment by cthoyt** [comment]
Thank you @JervenBolleman! Since one of the goals of this package is to remain 100% generic and well-tested, I would like to wait to finish the main parts before adding this in. Perhaps it might make sense to further subclass these components in the Bioregistry itself to get this functionality. We will for sure revisit this later.

**Comment by JervenBolleman** [comment]
> Thank you @JervenBolleman! Since one of the goals of this package is to remain 100% generic and well-tested, I would like to wait to finish the main parts before adding this in. Perhaps it might make sense to further subclass these components in the Bioregistry itself to get this functionality. We will for sure revisit this later.

Sounds very good to me. I just wanted to show the idea and then you can implement this when and if the time is right.

File: src/curies/mapping_service.py [hunk 1]
```diff
@@ -144,35 +146,34 @@ def __init__(
...
+                return
+
+            objects = [
+                URIRef(obj)
+                for obj in cast(Collection[str], self.converter.expand_pair_all(prefix, identifier))
+            ]
+            for obj, pred in itt.product(objects, self.predicates):
+                yield subj_query, pred, obj
+
+        if self.rdf_graph is not None:
```
[the remark this answers is not in the document]
Comment (JervenBolleman) [reply src/curies/mapping_service.py:174]: Return no data gracefully if the patterns don't match is how I would do it.

**Comment by cthoyt** [comment]
@JervenBolleman sorry I didn't mean to dismiss this, but since it was pointed at another branch I merged, it seems I can't directly reconcile it. I will for sure come back to this in the future

**Pull request closed without merging**